internal/project/framework/internals/error/error_handler.go

```diff
@@ -1,12 +1,13 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package error
 
 import (
 	"net/http"
 	"strings"
 
-	"{{ .PkgPath }}/framework/internals/sentry"
-	"{{ .PkgPath }}/framework/internals/validator"
+	/**#bean*/
+	"demo/framework/internals/sentry" /*#bean.replace("{{ .PkgPath }}/framework/internals/sentry")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines